### PR TITLE
Add breadcrumb to admin manage solution

### PR DESCRIPTION
**Pull request by NickNewlands**
Task: AB#15453
Story: AB#14712

### CODE REVIEW

**Comment by NickNewlands** [comment]
I need to change this to use the admin layout, just realised that it exists.

File: src/NHSD.GPIT.BuyingCatalogue.WebApp/Areas/Admin/Models/DetailsModel.cs [hunk 1]
```diff
@@ -30,5 +30,13 @@ public Address OrganisationAddress
         {
             get { return Organisation.GetAddress(); }
         }
+
+        public Dictionary<string, string> BreadCrumb()
+        {
+            return new Dictionary<string, string> {
```
Comment (leerogers-bjss) [inline src/NHSD.GPIT.BuyingCatalogue.WebApp/Areas/Admin/Models/DetailsModel.cs:36]: the ending brace needs to be on a new line

File: src/NHSD.GPIT.BuyingCatalogue.WebApp/Areas/Admin/Views/Organisations/Details.cshtml [hunk 2]
```diff
@@ -4,9 +4,11 @@
     ViewData["Title"] = "Buying Catalogue - Admin";
 }
 
+@section Breadcrumbs {
```
Comment (leerogers-bjss) [inline src/NHSD.GPIT.BuyingCatalogue.WebApp/Areas/Admin/Views/Organisations/Details.cshtml:7]: brace needs to be on a new line

**Review by leerogers-bjss — CHANGES_REQUESTED** [review]
you've got a failing unit test, fix this and the other minor style stuff and i'll approve it :)

```  Failed Get_CapabilitiesSelector_ReturnsDefaultView [62 ms]
  Error Message:
     Expected string length 24 but was 7. Strings differ at index 7.
  Expected: "Go back to previous page"
  But was:  "Go back"
  ------------------^

  Stack Trace:
     at NHSD.GPIT.BuyingCatalogue.WebApp.UnitTests.Areas.Solutions.Controllers.FuturesControllerTests.Get_CapabilitiesSelector_ReturnsDefaultView() in /home/vsts/work/1/s/tests/NHSD.GPIT.BuyingCatalogue.WebApp.UnitTests/Areas/Solutions/Controllers/FuturesControllerTests.cs:line 94
   at NUnit.Framework.Internal.TaskAwaitAdapter.GenericAdapter`1.GetResult()
   at NUnit.Framework.Internal.AsyncToSyncAdapter.Await(Func`1 invoke)
   at NUnit.Framework.Internal.Commands.TestMethodCommand.RunTestMethod(TestExecutionContext context)
   at NUnit.Framework.Internal.Commands.TestMethodCommand.Execute(TestExecutionContext context)
   at NUnit.Framework.Internal.Execution.SimpleWorkItem.<>c__DisplayClass4_0.<PerformWork>b__0()
   at NUnit.Framework.Internal.ContextUtils.<>c__DisplayClass1_0`1.<DoIsolated>b__0(Object _)```

File: src/NHSD.GPIT.BuyingCatalogue.WebApp/Areas/Admin/Models/DetailsModel.cs [hunk 3]
```diff
@@ -30,5 +31,22 @@ public Address OrganisationAddress
...
+                new NhsBreadcrumbModel {
+                    Controller = "admin",
+                    Action = nameof(HomeController.Index),
+                    Text = "Home",
+                },
+                new NhsBreadcrumbModel {
+                    Controller = "admin",
+                    Action = "buyer-organisations",
+                    Text = "Manage Catalogue Solutions",
+                },
```
Comment (NickNewlands) [inline src/NHSD.GPIT.BuyingCatalogue.WebApp/Areas/Admin/Models/DetailsModel.cs:48]: This sucks and needs improving, I could not see any amazing way to faithfully extract the 'route' from the controller, perhaps we need to setup a base class which marries up and can easily provides it? Something Like that. @leerogers-bjss

**Review by NickNewlands — COMMENTED** [review]
I pulled my other PR into this as I needed something from it (feature/14685-admin-manage-organisations).

**Pull request closed without merging**